lib/routes/utilities/pushReplicationMetric.js

```diff
@@ -0,0 +1,33 @@
+const assert = require('assert');
+
+const { ObjectMD } = require('arsenal').models;
+const { pushMetric } = require('../../utapi/utilities');
+
+function shouldPushMetric(objectMD) {
+    if (objectMD.getReplicationStatus() !== 'REPLICA') {
+        return false;
+    }
+    try {
+        const newObjectMD = new ObjectMD();
+        assert.deepStrictEqual(objectMD.getAcl(), newObjectMD.getAcl());
```

Comment:
Could be worth commenting both that comparing against a pristine object is to check that there is no tag nor ACL present in the object, as well as explaining why we don't want to push metrics in such case. I'm suspecting this is to distinguish normal PUT from PUT_TAGGING&co, in any case it does not seem obvious and deserves a comment I think.

If my assumption applies, wondering if it is possible to end up in a situation where we could count the same object twice, say by putting an empty tag set or ACL, or by putting then clearing them afterwards (not with DELETE_TAGGING but putting an empty set, for example) in which case it would be a problem to solve.

Reply:
Nice catch, thank you. I've added a comment and instead compared against the previous object metadata. That way we catch any case in which the ACL or tagging is changing (i.e. it is being replicated).